ingestion/src/metadata/ingestion/processor/pii.py

```diff
@@ -0,0 +1,201 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Processor util to fetch pii sensitive columns
+"""
+import logging
+import re
+import traceback
+from enum import Enum, auto
+
+from presidio_analyzer import AnalyzerEngine
+
+from metadata.generated.schema.entity.data.table import Table, TableData
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+
+PII = "PII"
+
+
+class PiiTypes(Enum):
+    """PiiTypes enumerates the different types of PII data"""
```

Comment:
these come from presidio?

Reply:
So we are converting from `presidio` to our PII Types and based on this we determine pii sensitive/nosensitive